airbyte-config/config-persistence/src/main/java/io/airbyte/config/persistence/ConfigRepository.java

```diff
@@ -1602,12 +1603,15 @@ public boolean getConnectionHasAlphaOrBetaConnector(final UUID connectionId) thr
     return countResult > 0;
   }
 
+  private static final List<Field<?>> baseConnectorBuilderProjectColumns =
```

Comment:
nit: should move to the top of the file and use CAPITAL_CASE format for static final object.

Reply:
done